pkg/convert/go_test.go

```diff
@@ -127,10 +107,92 @@ func TestFromGo(t *testing.T) {
 			}
 		]
 	}
-	`)
+	`),
+			expected: model.Translation{
+				Language: language.English,
+				Original: false,
+				Messages: []model.Message{
+					{
+						ID:          "1",
+						Message:     "{translatedMessage1}",
+						Description: "description1",
+						Positions:   []string{"src/config.go:10"},
+						Status:      model.MessageStatusFuzzy,
+					},
+					{
+						ID:          "2",
+						Message:     "{translatedMessage2}",
+						Description: "description2",
+						Positions:   []string{"src/config.go:20"},
+						Status:      model.MessageStatusFuzzy,
+					},
+					{
+						ID:          "3",
+						Message:     "",
```

Comment:
FromGo "message": "message3" to our message structure results In Message being empty?